src/redux/reducers/backendStatus/backendStatusLoaded.js

```diff
@@ -1,21 +1,13 @@
 /* eslint-disable no-param-reassign */
-import produce, { current } from 'immer';
+import produce from 'immer';
 
 import initialState from './initialState';
 
 const backendStatusLoaded = produce((draft, action) => {
   const { experimentId, status } = action.payload;
 
-  const previousStatus = current(draft)[experimentId]?.status;
-
-  const newStatus = {
-    pipeline: status.pipeline ?? previousStatus?.pipeline,
-    gem2s: status.gem2s ?? previousStatus?.gem2s,
-    worker: status.worker ?? previousStatus?.worker,
-  };
-
   draft[experimentId] = {
-    status: newStatus,
+    status,
```

Comment:
Do we still need this `status` variable even if we don't use this reducer to update the status anymore?

Reply:
We do use it, we just update all the status instead of only one part of it. If you check the api, you can see the response to getBackendStatus returns an update on all the pipelines + worker.
<img width="603" alt="image" src="https://user-images.githubusercontent.com/8465903/156356414-85829a8a-30ad-472a-afd3-ea78bf097ddc.png">